runtime/kernel/dma/blue_dma.c

```diff
@@ -39,8 +39,7 @@
 irqreturn_t blue_dma_intr_handler_read(int irq, void * dev_id)
 {
 	struct dma_engine *dma = (struct dma_engine *)dev_id;
-	struct platform *p = &dma->dev->cls->platform;
-	volatile uint32_t* msix_ack = (volatile uint32_t*) (dma->dev->mmap.plat + ((0x500000 + 0x8120) - p->plat.base));
+	volatile uint32_t* msix_ack = (volatile uint32_t*) (dma->dev->mmap.plat + 0x28120);
```

Comment:
Are these addresses supposed to stay hardcoded?

Reply:
No, in fact they are not! Good catch

Reply:
Solved that issue in 68df96a